packages/react-urql/package.json

```diff
@@ -60,6 +61,7 @@
   },
   "dependencies": {
     "@urql/core": "^2.3.6",
+    "use-sync-external-store": "^1.0.0-rc.0",
```

Comment:
I'm wondering; when is this releasable given that this package is still tagged as an `rc`? Would we want to define `^1.0.0-rc.0 || ^1.0.0`?